src/pages/Dogs/AdoptionLogCard.jsx

```diff
@@ -0,0 +1,270 @@
+            // p="4"
+            rounded="md"
+            bg="white"
+            margin="30px 0px"
+            width="280px"
+            // overflow="hidden"
+            // maxW="sm"
+            justifyContent="flex-end"
+            key={props.key}
+          >
+            <CardHeader borderBottom="1px solid #CBD5E0">
+              <Flex direction="row">
+                <Flex alignItems="center" flexWrap="nowrap" direction="row">
+                  <Avatar
+                    marginRight="10px"
+                    size="lg"
+                    src="https://ca-times.brightspotcdn.com/dims4/default/30a8879/2147483647/strip/false/crop/2048x1152+0+0/resize/1486x836!/quality/80/?url=https%3A%2F%2Fcalifornia-times-brightspot.s3.amazonaws.com%2F5a%2Fc5%2F10a8781f4130240d4b4d42d12794%2Fla-1489021677-ia7m9d0xvx-snap-photo"
+                  />
+                  <Box maxWidth="100%" maxHeight="100%">
+                    <Flex flexDirection="row">
+                      <Heading fontSize="xl">{dogName}</Heading>
+                      <Checkbox></Checkbox>
+                    </Flex>
+                    <Text size="lg" marginRight="10px">
+                      aka {setAltName()}
+                    </Text>
+                    <Text size="lg" marginRight="10px">
+                      Grad Age: {gradAge}
+                    </Text>
+                    <Text size="lg" marginRight="10px">
+                      Breed: {breed}
+                    </Text>
+                  </Box>
+                </Flex>
+              </Flex>
+              <Flex gap="5px" flexWrap="wrap">
+                {service && (
+                  <Tag background="#48BB78" color="#FFFFFF">
+                    Service
+                  </Tag>
+                )}
+                {therapy && (
+                  <Tag background="#4299E1" color="#FFFFFF">
+                    Therapy
+                  </Tag>
+                )}
+                {staffAdoption && (
+                  <Tag background="#ECC94B" color="#FFFFFF">
+                    Stf Adpt
+                  </Tag>
+                )}
+                {specialNeeds && (
+                  <Tag background="#ED8936" color="#FFFFFF">
+                    Special
+                  </Tag>
+                )}
+                {deceased && (
+                  <Tag background="#C53030" color="#FFFFFF">
+                    Decsd
+                  </Tag>
+                )}
+              </Flex>
+            </CardHeader>
+            <CardBody>
+              <Flex gap={2} flexDirection="column">
+                <Box>
+                  <Flex gap="3px">
+                    <Text>Facility: </Text>
+                    <Text>{facility}</Text>
+                  </Flex>
+                </Box>
+                <Box>
+                  <Text as="b">Adopter</Text>
+                  <Text>Name: {adopter}</Text>
+                  <Text>
+                    Phone: <a>{phoneNumber}</a>
```

Comment:
🚫 **[eslint]** <[jsx-a11y/anchor-is-valid](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/anchor-is-valid.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md